lib/src/components/NxLoadWrapper/__tests__/NxLoadWrapper.test.tsx

```diff
@@ -4,48 +4,77 @@
-  });
+    it('retryHandler is called when that Retry button is clicked', async ()=> {
+      const user = userEvent.setup();
+      const retryHandler = jest.fn();
+      expect(renderEl()?.textContent).not.toContain('Retry');
+      expect(quickRender({ error: 'Error', retryHandler }).getByRole('button', { name: 'Retry' }))
+          .toBeInTheDocument();
+      const retryButton = screen.getByRole('button', { name: 'Retry' });
+      expect(retryHandler).not.toHaveBeenCalled();
+      await user.click(retryButton);
+      expect(retryHandler).toHaveBeenCalled();
+    });
+
+    it('children content and loading spinner are not rendered when error is present', ()=> {
+      const toCheck = ['Loading…', 'Foo'];
+      toCheck.forEach(nameToCheck => {
+        expect(renderEl({error: 'Error'})?.textContent).not.toContain(nameToCheck);
+      });
 
-  it('renders a loading spinner if error is not set and loading is true', function() {
-    const SpinnerFixture = () => <NxLoadingSpinner />;
+    });
 
-    expect(getShallowComponent({ loading: true })).toMatchElement(<SpinnerFixture/>);
-    expect(getShallowComponent({ error: 'foo', loading: true })).not.toMatchElement(<SpinnerFixture/>);
   });
 
-  it('renders provided children if loading is false and error is unset', function() {
-    expect(getShallowComponent()).toContainReact(children);
-    expect(getShallowComponent({ loading: false })).toContainReact(children);
-    expect(getShallowComponent({ error: 'foo' })).not.toContainReact(children);
-    expect(getShallowComponent({ loading: true })).not.toContainReact(children);
+  describe('when it is loading', ()=> {
+    expect(renderEl()?.textContent).not.toContain('Loading…');
+    expect(quickRender({ loading: true }).container.textContent).toContain('Loading…');
   });
 
-  it('renders children provided by a function if loading is false and error is unset', function() {
-    const childrenFn = () => children;
+  describe('renders provided children', ()=> {
+    it('renders provided children if loading is false and error is unset', ()=> {
+      const { container: withChildren } = quickRender();
+      expect(withChildren.textContent).toContain('Foo');
+    });
 
-    expect(getShallowComponent({ children: childrenFn })).toContainReact(children);
-    expect(getShallowComponent({ children: childrenFn, loading: false })).toContainReact(children);
-    expect(getShallowComponent({ children: childrenFn, error: 'foo' })).not.toContainReact(children);
-    expect(getShallowComponent({ children: childrenFn, loading: true })).not.toContainReact(children);
+    it('renders children provided by a function if loading is false and error is unset', ()=> {
+      const childrenFn = () => children;
+      const { container: withFuncChild } = quickRender({
+        children: childrenFn,
+        loading: false,
+        error: ''
+      });
+      expect(withFuncChild.textContent).toContain('Foo');
+    });
   });
```

Comment:
Also test that loading spinner and error alert are not present.

Reply:
> Also test that loading spinner and error alert are not present.

Done c89e56f0b518e44e1cc81823b8093f6a35b9c743